src/components/Autocomplete/index.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { ReactSearchAutocomplete } from 'react-search-autocomplete';
+import { useFetch } from '../../hooks/useFetch';
+
+const Autocomplete = ({ setCity, }) => {
+	const [locations, setLocations] = useState([]);
+
+	const [{ data }, fetchData] = useFetch();
+
+	const fetchAutocompleteResults = (city) => {
+		if (city) {
+			fetchData(`https://autocomplete.search.hereapi.com/v1/autocomplete?q=${city}&limit=4&types=city&apiKey=${process.env.REACT_APP_HERE_APIKEY}`);
+		}
+	};
+
+	useEffect(() => {
+		let cleanedLocations = [];
+		if (data?.items) {
+			let i = 0;
+			cleanedLocations = data.items.map((item) => {
+				return { id: i++, name: item.address.label };
+			});
+		}
+		setLocations(cleanedLocations);
+	}, [data]);
+
+	const searchAutocomplete = (city) => {
+		fetchAutocompleteResults(city);
+	};
+
+	const handleSelect = (label) => {
+		let city = label.name.split(',')[0];
```

Comment:
Can we use 'const' here instead of 'let'?

Reply:
Done